src/pcm.c

```diff
@@ -650,6 +654,12 @@ static int pcm_hw_mmap_status(struct pcm *pcm)
         goto mmap_error;
     }
 
+    if (pcm->config.avail_min) {
+        pcm->mmap_control->avail_min = pcm->config.avail_min;
+    } else {
+        pcm->mmap_control->avail_min = 1;
+    }
```

Comment:
I'm curious who is using `pcm->mmap_control->avail_min` and why we need to set this variable. When we sync the mmap_control, will it overwrite the variable in the ALSA kernel driver?

Reply:
Driver will not return avail_min, that's why set here. avail_min is configured by user or ALSA itself.

Reply:
Since we set `pcm->config.avail_min` in `pcm_set_config`, I think we can just copy it.